azure-iot-device/azure/iot/device/iothub/config.py

```diff
@@ -0,0 +1,29 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
```

Comment:
Still think this should be in the pipeline package

Reply:
moved to pipeline

---
In reply to: [337672921](https://github.com/Azure/azure-iot-sdk-python/pull/368#discussion_r337672921) [](ancestors = 337672921)